aequilibrae/project/project.py

```diff
@@ -82,35 +92,35 @@ def open(self, project_path: str) -> None:
 
     @property
     def project_base_path(self):
-        return self.__base_path
+        return self.scenario.base_path  # do we want to use the root scenario here?
```

Comment:
In response to the comment, I am happy to keep the scenario path here. We do have the root_scenario always available.

Maybe we should rename _project_base_path_ to _base_path_ or maybe even keep _project_base_path_ as the actual root directory

Reply:
Can you add this as an issue for AequilibraE 2.0, @Jake-Moss